### DISCUSSION TITLE
"Ghost" Jekyll theme after deleting _config.yml

### DISCUSSION THREAD

**Discussion by landonma** [community/community#22343] [Other Feature Feedback, Questions, & Ideas]
<p>When I upload a _config.yml containing something like this “theme: jekyll-theme-architect” then delete it, my markdown file (index.md) are still able to be accessed and some sort of jekyll formating seems to presit. Without “intializing” jekyll in this way (created then deleting the file) the index file is not accesable. I like this basic formating and would like to contain a _config.yml file or something of the sort so people who clone my repository and make their own will automotically have this formatting type and have jekyll going without having to create and delete the file. </p>
<p>As far as I can tell, it does not matter which jeykll theme you start with then delete and if you upload “theme: jekyll” in the _config.yml file it gives you unformatted text. Uploading a blank _config.yml does not activate jekyll. </p>
<p>unformatted text: <a href="https://landonma.github.io/Data-Management-Internal-Sharing-Creator/" rel="nofollow noopener">https://landonma.github.io/Data-Management-Internal-Sharing-Creator/</a></p>
<p>ghost theme: <a href="https://landonma.github.io/datamanagement/" rel="nofollow noopener">https://landonma.github.io/datamanagement/</a></p>
<p>I don’t know anything about jekyll really so any insite into what is going on would be appreated. </p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/landonma">@landonma</a>,</p>
<p>You can get this “ghost theme” by <a href="https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/#enabling-github-pages-to-publish-your-site-from-master-or-gh-pages" rel="nofollow noopener">activating GitHub Pages for your repository</a> without selecting a theme in the theme selection dialog box. This doesn’t involve creating or deleting a _config.yml file.</p>
<p>Hope this helps!</p>

**Comment by landonma** [comment]
<p>Blast! My impatants undoes me again. Thanks, I thought I tried this but I just didn’t wait long enought. 100% want I wanted. </p>
